our-app/src/components/unauthenticated/HomePage.js

```diff
@@ -1,9 +1,82 @@
 import React from 'react';
+import { Carousel } from 'react-bootstrap';
 
-function HomePage() {
+export default function HomePage() {
+    const mystyle = {
+        color: "Black",
+        fontFamily: "Helvetica"
+    };
     return (
-        <h1>WELCOME</h1>
+        <div>
+            <h1 style={mystyle}><b>Welcome to Health Wallet</b></h1>
+            <Carousel>
+                <Carousel.Item>
+                    <img
+                        width={800}
+                        height={450}
+                        alt="Welcome to Health Wallet"
+                        src="/server/immunization.jpg"
+                    />
+                </Carousel.Item>
```

Comment:
We could make this more compact by using the map function on an array of src strings as following:

```
const imgSources= [
"/server/immunization.jpg",
"/server/immunization1.jpg",
"/server/immunization2.jpg",
"/server/immunization3.jpg",
"/server/immunization4.jpg",
"/server/immunization5.jpg",
"/server/immunization6.jpg",
"/server/immunization7.jpg"
];

const items= imgSources.map((source) =>
   <Carousel.Item>
                    <img
                        width={800}
                        height={450}
                        alt="Welcome to Health Wallet"
                        src={source}
                    />
   </Carousel.Item>
);

<Carousel>
        {items}
<Carousel/>
```

Let me know if this makes sense!

Reply:
You will also need to add a key on your mapped items

Reply:
My bad with Package.json. I think that might have been due to some packages that I added but then ended up not using them. I thought I was able to discard the changes in our original packages.json but I will probably have to go back and update my branch to the remote. 

Also, thanks for the feedback. I think you are right with the code not being compact right now. I will have to reformat the whole thing I wrote. I will probably run through you first before creating a pull request. Perhaps we can have a pair programming session and I can share my screen with you when I have something solid to show you. Let me know what you think?

Reply:
I'm down to collab through google meet whenever! Just let me know.